new-pipeline/src/evo.js

```diff
@@ -337,6 +337,13 @@ $(function () {
               var metric = $("#measure").val()
               var channel = $("#max-channel-version").val().split("/")[0]
               var description = getDescriptionWithLink(metric, channel, evolutionDescription)
+              var improvedDescription = document.createElement("button");
```

Comment:
Let's just keep the changes inside `getDescriptionWithLink()` in dashboards.js for now.

Reply:
sure :+1: I'll transfer all of this there only